src/core/device.c

```diff
@@ -165,6 +165,7 @@ static int device_serialize(Unit *u, FILE *f, FDSet *fds) {
         assert(fds);
 
         unit_serialize_item(u, f, "state", device_state_to_string(d->state));
+        unit_serialize_item_format(u, f, "found", "%i", d->found);
```

Comment:
i feel a bit nervous about serializing the binary value of this flag field, as it turns something into an API that is not necessarily obvious as one... Hence, I'd rather see three individual bool fields serialized for the three relevant bits in the flags field...

Reply:
good point, I'll update the PR accordingly, thanks for the review.